src/main/java/com/github/cypher/sdk/Room.java

```diff
@@ -397,13 +397,19 @@ private void parseMemberEvent(JsonObject event, int originServerTs, String sende
 				User user = userRepository.get(memberId);
 				user.update(event);
 
+				Member member = new Member(user);
+
 				if        ("join".equals(membership)) {
```

Comment:
Formatting looks weird (unintended?).

Reply:
That's not part of the PR, although I do have strange ideas about alignment.